src/disco/reducers/telemetry.js

```diff
@@ -0,0 +1,46 @@
+/* @flow */
+import invariant from 'invariant';
+
+export const SET_HASHED_CLIENT_ID: 'SET_HASHED_CLIENT_ID' =
+  'SET_HASHED_CLIENT_ID';
+
+export type TelemetryState = {|
+  hashedClientId?: string,
```

Comment:
Maybe we could make it non-optional and nullable?

Reply:
I personally prefer undefined if it's not set but I can default the state to null.